lock-keeper-client/src/client.rs

```diff
@@ -208,11 +220,18 @@ impl LockKeeperClient {
     pub(crate) async fn create_channel(
         client: &mut LockKeeperRpcClient<LockKeeperRpcClientInner>,
         action: ClientAction,
+        account_name: &AccountName,
     ) -> Result<ClientChannel, LockKeeperClientError> {
         // Create channel to send messages to server after connection is established via
         // RPC
         let (tx, rx) = mpsc::channel(2);
-        let stream = ReceiverStream::new(rx);
+        let mut stream = Request::new(ReceiverStream::new(rx));
+
+        let account_name_val = MetadataValue::try_from(account_name.to_string())
+            .map_err(|_| LockKeeperClientError::InvalidAccount)?;
```

Comment:
```suggestion
        let account_name_val = MetadataValue::try_from(account_name.to_string())?;
```

This tonic error should be added to the error type. This should work: 
```rust
    #[error(transparent)]
    TonicMetadata(#[from] tonic::metadata::errors::InvalidMetadataValue),
```